### ISSUE TITLE
converting tensorflow pb or tflite

### ISSUE THREAD

**Issue by jnorwood** [KhronosGroup/NNEF-Tools#43]
This page has a several tar'd up training models and eval models for tensorflow in protobuf,  tflite and pbtext forms.  They also have saved checkpoint data.   These seem closer to what was used for conversion for the caffe converter.  

https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet_v1.md

They may have come from these tf.slim models at the link below, but that isn't clear, and in any case the tf.slim code doesn't look anything like the simple low level tf.nn calls provided in the tf_to_nnef small_net() conversion example.
https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet/mobilenet.py

I think, in general, there will be models available for download in these pb and tflite formats, without the available python code that is currently required for the conversion program.

There are tflite flatbuf and tf protobuf parser implementations at this link that are using the generated schemas for the tensorflow .
https://github.com/ARM-software/armnn/tree/master/src/armnnTfParser

It seems to me the protobuf and tflite persisted  models and weights would be a more useful strarting point for the conversion flow.  Was there some reason they were not used?
Thanks.

**Comment by gyenesvi** [comment]
We are actually working on another TF converter that starts from protobuf input, just as you say. The current converter that starts from the Python code has the advantage that it is possible to easily catch a group of operations (defined as a single function in Python) and convert them as one higher level compound. For example it was able to export batch norm as a single op before fused_batch_norm was introduced.

**Comment by jnorwood** [comment]
You can also use the flatbuffer compiler, flatc, to dump the tflite  to json, if you provide the schema.
flatc -t schema.fbs -- input_model.tflite
I'm looking at the MobileNet_v1_1.0_224_quant.tflite json dump and it appears they do represent subgraphs in the model.  For example, BiasAdd,  SpatialSqueeze, Relu6 are in subgraph sections that aren't being displayed by Netron currently.

**Comment by jnorwood** [comment]
After looking at the json more, it isn't clear if the subgraph section is supposed to provide sub-operations of higher level layers, or if it is just providing tensor data that was collected from the range measurements during training .... operations which have been deleted from the inference graph.
